addons/mrp/migrations/11.0.2.0/post-migration.py

```diff
@@ -197,12 +197,6 @@ def fill_stock_move_line_consume_rel(cr):
             ON sml1.move_id = sqmr1.move_id
         INNER JOIN stock_move_line sml2
             ON sml2.move_id = sqmr2.move_id
-        WHERE NOT EXISTS (
```

Comment:
You can replace with `ON CONFLICT DO NOTHING`

Reply:
Which PG version introduces this syntax? Maybe the version is too high for this Odoo version

Reply:
```insert into res_groups_users_rel values(2, 3);
ERROR:  duplicate key value violates unique constraint "res_groups_users_rel_pkey"
DETAIL:  Key (gid, uid)=(2, 3) already exists.
insert into res_groups_users_rel values(2, 3) ON CONFLICT DO NOTHING;
INSERT 0 0
```

Reply:
It was introduced in PG 9.5:

https://wiki.postgresql.org/wiki/What%27s_new_in_PostgreSQL_9.5#INSERT_..._ON_CONFLICT_DO_NOTHING.2FUPDATE_.28.22UPSERT.22.29

What is the minimum PG version for v11?

Reply:
It's introduced in 9.5

Reply:
I think that's entirely reasonable to expect people to run their migration on. Our travis config uses 9.6.

Reply:
OK, no problem for me.